tasmota/tasmota_xsns_sensor/xsns_23_me007.ino

```diff
@@ -0,0 +1,535 @@
+/*
+  xsns_23_me007ula.ino - ME007 ultrasonic sensor support for Tasmota
+
+  Copyright (C) 2022  Mathias Buder
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifdef USE_ME007
+/*********************************************************************************************\
+ * ME007 - Ultrasonic distance sensor
+ *
+ * Code for ME007 family of ultrasonic distance sensors
+ * References:
+ * - https://wiki.dfrobot.com/Water-proof%20Ultrasonic%20Sensor%20(ULS)%20%20SKU:%20SEN0300
+\*********************************************************************************************/
+
+/*********************************************************************************************/
+/* Includes*/
+/*********************************************************************************************/
+#include <TasmotaSerial.h>
+
+/*********************************************************************************************/
+/* Defines */
+/*********************************************************************************************/
+#define XSNS_23                            23
+
+#define ME007_VERSION                      "1.0.0"                                              /**< Driver version X.Y.Z: X:Major, Y: Minor, Z: Patch */
+
+#define ME007_DEBUG_MSG_TAG                "ME007: "
```

Comment:
Pls change to a three character tag like "ME7: "

Reply:
Ok, done!